app/src/main/java/com/example/footprnt/Map/MapFragment.java

```diff
@@ -463,7 +463,7 @@ public void onClick(View v) {
                 mSwipe.start();                 // Add sound when user sends post
                 final String title = ((EditText) mAlertDialog.findViewById(R.id.etTitle)).getText().toString();
                 final String snippet = ((EditText) mAlertDialog.findViewById(R.id.etSnippet)).getText().toString();
-                if( TextUtils.isEmpty(title) || TextUtils.isEmpty(snippet)){
+                if( TextUtils.isEmpty(title) || TextUtils.isEmpty(snippet)) {
```

Comment:
Why a space at the start of the paranthesis?

if(TextUtils.isEmpty(title) || TextUtils.isEmpty(snippet)) {